pkg/utils/redis_utils.go

```diff
@@ -14,9 +14,25 @@ import (
 
 const RedisConnTimeoutSecond = 1
 
+func newRedisConn(addr string, connectTimeout time.Duration, readTimeout time.Duration, writeTimeout time.Duration, password string) (redis.Conn, error) {
+	c, err := redis.DialTimeout("tcp", addr, connectTimeout, readTimeout, writeTimeout)
```

Comment:
I think we should use RedisConnTimeoutSecond in newRedisConn directly

Old codes use 1s timeout totally.